### PR TITLE
photos and emojis open project prototype

### PR DESCRIPTION
**Pull request by rebeccaruva**
What was added to _chat.jsp_:
- __chevron arrow__ to explore example shared photos page
- __camera button__ gives alert message
- divs to make message text box and camera button sit next to each other

_chat-photos.jsp_ is an example skeleton for what the shared photos page will look like. For now it looks more like one page that can hold all photos from all conversations, but ideally I would like to set up individual pages for each chat (just how chat pages are set up). As for the photos on the page, they are set up to be in dual columns with divs.

### CODE REVIEW

**Comment by jdtyner** [comment]
Looks good, but I noticed a few things:

Clicking the back button (the chevron that you put in, not the browser's back button) from the photos page returns to the general conversations page, and not the specific conversation the user was in before.

The camera icon [overlaps the text box](https://i.imgur.com/Xl4aUPu.png) for me on Safari. It looks better on Chrome. I don't know if this is a Safari problem or a bug in your html, but I'm just telling you in case you wanna look into it.

Also, I think the right chevron arrow that goes to the photos page is a bit ambiguous. I think text saying "photos" next to the button would make it more clear.

File: src/main/java/codeu/controller/PhotosServlet.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet class responsible for the chat page. */
+public class PhotosServlet extends HttpServlet {
```
Comment (srjoglekar246) [inline src/main/java/codeu/controller/PhotosServlet.java]: Tests? Some things you could test:
1. Whether doGet redirects properly based on conversation title (including edge cases such as invalid titles, empty string)
2. Whether doPost redirects to doGet
3. What happens when unsigned user tries going to the link?

File: src/main/webapp/WEB-INF/view/chat.jsp [hunk 2]
```diff
@@ -42,6 +51,29 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       var chatDiv = document.getElementById('chat');
       chatDiv.scrollTop = chatDiv.scrollHeight;
     };
+
+    // When the user clicks the button, open the modal
+    function showModal() {
+      // Get the modal
```
Comment (srjoglekar246) [inline src/main/webapp/WEB-INF/view/chat.jsp]: Redundant comment. Remove

**Comment by srjoglekar246** [comment]
Fix conflicts before merging :-). Also, +1 to Alisha's comment on the link.
